### PR TITLE
Fix various Dark theme issues

### PR DESCRIPTION
**Pull request by MootezSaaD**
Related issues: #5522 , #5699 

- [X] Drag'n'drop text

image

- [X] Dialogs
image

image

image

The issue is that I don't find the dialogs graphics (e.g. the confirmation image) adequate, so maybe create custom ones for the dark theme (or remove them)?

- [X] Scroll pane
The scroll pane is quite visible (at least on my side) so I don't know if that issue has been considered as resolved or not?

image

- [ ] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [X] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/gui/JabRefDialogService.java [hunk 1]
```diff
@@ -126,6 +127,7 @@ public static String shortenDialogMessage(String dialogMessage) {
         choiceDialog.setHeaderText(title);
         choiceDialog.setTitle(title);
         choiceDialog.setContentText(content);
+        Globals.getThemeLoader().installCss(choiceDialog.getDialogPane().getScene(), Globals.prefs);
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/gui/JabRefDialogService.java:130]: We are trying to remove the `Globals` class (in the long term). To facilitate this, could you please extract the `themeLoader` and `preferences`, and initialize them by passing the global objects as dependencies through the constructor (i.e. move `Globals` from this class to the caller)

File: src/main/java/org/jabref/gui/JabRefDialogService.java [hunk 2]
```diff
@@ -148,12 +150,14 @@ public static String shortenDialogMessage(String dialogMessage) {
     @Override
     public void showInformationDialogAndWait(String title, String content) {
         FXDialog alert = createDialog(AlertType.INFORMATION, title, content);
+        Globals.getThemeLoader().installCss(alert.getDialogPane().getScene(), Globals.prefs);
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/JabRefDialogService.java:153]: Does it work, if you move this line inside the createDialog? Then you could reduce some code duplication

Comment (MootezSaaD) [reply src/main/java/org/jabref/gui/JabRefDialogService.java]: Yes! didn't really pay attention.

File: src/main/java/org/jabref/gui/JabRefDialogService.java (hunk 1 above)
Comment (MootezSaaD) [reply src/main/java/org/jabref/gui/JabRefDialogService.java]: I have refactored the code but I m not sure if I've done it correctly

**Comment by MootezSaaD** [comment]
Changes regarding the scroll bar:

image

On hover:
image

**Comment by tobiasdiez** [comment]
Perfect. Thank you again!

**Pull request merged**